src/main/java/bio/terra/service/filedata/google/gcs/GcsPdao.java

```diff
@@ -159,17 +164,17 @@ public void deleteFile(FireStoreFile fireStoreFile) {
     private boolean deleteWorker(GoogleBucketResource bucketResource, String bucketPath) {
         GcsProject gcsProject = gcsProjectFactory.get(bucketResource.getProjectResource().getGoogleProjectId());
         Storage storage = gcsProject.getStorage();
-        Optional<Blob> blob = Optional.ofNullable(storage.get(BlobId.of(bucketResource.getName(), bucketPath)));
-        if (blob.isPresent()) {
-            return blob.get().delete();
+        Blob blob = storage.get(BlobId.of(bucketResource.getName(), bucketPath));
+        if (blob != null) {
+            return blob.delete();
```

Comment:
Any specific reason for the change here?

Reply:
My understanding for best practices for using `Optional` is only to use it as a result type from a method. My opinion is that constructing a new object just to be able to avoid doing a comparison to null is wasteful and makes the code less readable.